airflow/upgrade/rules/logging_configuration.py

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import absolute_import
+
+from airflow.configuration import conf
+from airflow.upgrade.rules.base_rule import BaseRule
+
+
+class LoggingConfigurationRule(BaseRule):
+    title = "Logging configuration has been moved to new section"
+
+    description = "The logging configurations have been moved from [core] to the new [logging] section."
+
+    def check(self):
+        if not conf.has_option("logging", "logging_level"):
+            return "The logging configurations have been to moved from [core] to the new [logging] section."
```

Comment:
I think it would be good to tell users which keys should be moved

Reply:
Ah yes will add them